### DISCUSSION TITLE
Can't re-verify my academic affiliation

### DISCUSSION THREAD

**Discussion by Aarjav0210** [community/community#51878] [GitHub Education]
Hi there!

I received a coupon expiration reminder email, and the email said:

> If you are still a student or teacher, you may apply for a new academic discount coupon

So I followed the steps and tried to re-verify my academic affiliation. However, I received the following error message:

> - You are unlikely to be verified until you have completed your GitHub user profile with your full name as it appears in your academic affiliation document plus a short bio.
> - The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote.
> - Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy.
> - You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so.

I am currently studying in my year abroad. This is indicated on the form of proof I uploaded along with my up-to-date term-time address (which is not in the same country). I suspect my current location is throwing off the system, but I have provided all the information. The reminder email said my GitHub education coupon will expire on 08 April 2023 and my account will be downgraded. 

I hope this can be resolved soon to avoid the downgrading.

Thank you in advance.

**Comment by samus-aran** [comment]
Hi @Aarjav0210 

Let's see how we can assist ⭐ Thank for sharing your rejection reasons 👍 

Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

Often this warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

It would be preferable that a student is on campus when applying and thus able to upload a document but with remote learning that isn't always possible, so we request students submit a live photo of their evidence rather than an upload. We understand that students may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it. Here are a few tips on using a photo to capture.

1. Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor (a laptop perhaps - or a TV) rather than a phone.
2. If you can, print off your evidence and take a photo of this copy.
3. Keep the text legible. It's preferred to have the full shot of the evidence but try and capture the important sections of it to confirm your student status. If the photo is too large and makes the text tiny then zoom in appropriately.

Let us know here if you need further assistance

**Reply by ahmadhassan7** [reply]
@samus-aran I'm facing this issue too. I got 10 days left in my account expiry. 
I'm still a student and I tried re-verifying 6 times, got rejected every time.
Uploading Student ID straight didn't worked for me I was getting error message to use camera to upload image then rest of the 5 times I tried again by taking photo via my laptop's camera holding my Student ID and including my face too to ensure maximum transparency but still didn't worked. My Student ID clearly shows institution name and the dates which are necessary to be shown.
